features/FxA-107-security-event-history/README.md

```diff
@@ -70,5 +228,63 @@ we will log the `Account.history.seen` message.
   the the tides.
 
 - **Users may wish to revoke an address**: If a user has signed in while
-  in a hotel some where, they may wish to be able to to signal that this
+  in a hotel somewhere, they may wish to be able to signal that this
   IP is really **not** something to be trusted.
+
+
+
+## Results
+
+(This section to be completed once deployed to production)
```

Comment:
This is a new thing I'm trying out based on experience in the product meeting - stubbing out the results that we plan to insert once the feature gets to "measuring", so that it's obvious what the next steps are once we've shipped it.  So far I'm finding it very helpful to clarify whether we're shipping all the things to need to ship in order to judge success of a feature.